docs/api.md

```diff
@@ -429,7 +429,9 @@ Transformers are basically hooks for encoding and decoding records, which can wo
 
```

Comment:
"For now, only remote transformers have been implemented, but there are plans to implement local transformers and collection transformers in a next version." (and then remove the 'local transformers' section, see https://github.com/Kinto/kinto.js/pull/171/files#r40066296)